### PR TITLE
Merge 4.41 code freeze to develop

### PR DESCRIPTION
**Pull request by oguzkocer**

### CODE REVIEW

File: Simplenote/Resources/AppStoreStrings.pot
Comment (mokagio) [inline Simplenote/Resources/AppStoreStrings.pot]: This suggestion will need to be applied before merging, or the GlotPress update will fail

```suggestion
msgid "notes,cloud,simple,notebook,text,to-do,todo,planner,journal,diary,sync,notepad,tag,writing,memo"
```

File: Simplenote/Resources/release_notes.txt
Comment (mokagio) [inline Simplenote/Resources/release_notes.txt:1]: Looking at the `.pot`, I wonder why what looks like the same string in resulted in different formatting 🤔 The diff in this file is the answer 😄

**Review by mokagio — COMMENTED** [review]
- [x] New version header in `RELEASE-NOTE.txt` 
- [x] Version update in `.xcconfig`, `Deliverfile` (not for WP), and `download_metadata.swift` 
- [x] `Localizable.strings` updated 
- [x] `release_notes.txt` updated with notes from `RELEASE-NOTE.txt` for current version 
- [x] `AppStoreStrings.pot` updated accordingly to new version and release notes (for non-editorialized projects)

I'm not approving because of my suggestion and because auto-merge is enabled.

File: Simplenote/Resources/AppStoreStrings.pot
Comment (oguzkocer) [reply Simplenote/Resources/AppStoreStrings.pot]: @mokagio I think you mentioned there was a scripting issue or something about this. Can you remind me what it is?

Are we going to need to make this change every code freeze? 🤔 

Also, I need to update keywords again, so I'll wait for your answer and do that before we merge this in.

Comment (mokagio) [reply Simplenote/Resources/AppStoreStrings.pot]: I did, but I haven't had the time to track the issue down yet 😞 

Maybe @jkmassel knows? I think he bumped into it, too in the near past.

Comment (oguzkocer) [reply Simplenote/Resources/AppStoreStrings.pot]: @mokagio I've updated the `keywords` again in a2321612d764c0a167b6f2b9a5a501d6696d3f3b, ran `update_appstore_strings` in d0ffca0dc2b5b48019dae1b37815a72aae5f7bda and removed the extra newline in bae72340ef0dd64c2696d873b4dc28b2f0e6ab37. This is ready for another look!

**Review by mokagio — APPROVED** [review]
All looking good now 👍

**Pull request merged**
